### PR TITLE
Dynamic batching of embed/rerank on single gaudi without wrapper

### PR DESCRIPTION
**Pull request by Spycsh**
## Description

Add dynamic batching of embed/rerank on single gaudi without wrapper

depends on https://github.com/opea-project/GenAIComps/pull/774

## Issues

n/a

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)
- [ ] Others (enhancement, documentation, validation, etc.)

## Dependencies

None
## Tests

NA

### CODE REVIEW

File: ChatQnA/Dockerfile.no_wrapper_dynamic_batching
Comment (lvliang-intel) [inline ChatQnA/Dockerfile.no_wrapper_dynamic_batching:1]: no_wrapper is default, so please remove no_wrapper in the file naming.

File: ChatQnA/Dockerfile.dynamic_batching [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Copyright (C) 2024 Intel Corporation
```
Comment (chensuyue) [inline ChatQnA/Dockerfile.dynamic_batching:1]: Need to add into image build.

File: ChatQnA/docker_compose/intel/hpu/gaudi/compose_dynamic_batching.yaml [hunk 2]
```diff
@@ -0,0 +1,127 @@
+# Copyright (C) 2024 Intel Corporation
```
Comment (chensuyue) [inline ChatQnA/docker_compose/intel/hpu/gaudi/compose_dynamic_batching.yaml:1]: Need to have an e2e test for this compose.

File: ChatQnA/docker_compose/intel/hpu/gaudi/compose_dynamic_batching.yaml
Comment (lvliang-intel) [inline ChatQnA/docker_compose/intel/hpu/gaudi/compose_dynamic_batching.yaml:1]: The file name need to adapt for the application scenario from users' perspective.

**Comment by Spycsh** [comment]
@chensuyue @lvliang-intel Maybe we can just close this PR and I will keep the branch for reproduction in the future? I do not think there are any appropriate names to describe this. Also I think this feature is not mature enough for this release.

**Pull request closed without merging**